vulture/stripes_lib/stripes.py

```diff
@@ -121,17 +124,26 @@ def _check_location_is_near(self, lat, lon, point_ds, ds):
     
         Returns tuple of: (eastings, northings)
         """
-        y = float(point_ds.projection_y_coordinate.values)
-        x = float(point_ds.projection_x_coordinate.values)
+        y = float(point_ds.lon.values)
+        x = float(point_ds.lat.values)
         
-        lat_diff = abs(float(ds.latitude.sel(projection_y_coordinate=y, projection_x_coordinate=x)) - lat)
-        lon_diff = abs(float(ds.longitude.sel(projection_y_coordinate=y, projection_x_coordinate=x)) - lon)
+        lat_diff = abs(float(ds.lat.sel(lon=y, lat=x)) - lat)
```

Comment:
Since lat and lon are now 1D arrays, you shouldn't need to select them using `lon` and `lat` - only the coordinate that is relevant in either x or y.